docs/README.md

```diff
@@ -1,85 +1,95 @@
 # Lockbook
 
+[<img height= "30" src="https://apple-resources.s3.amazonaws.com/media-badges/download-on-the-app-store/black/en-us.svg">](https://apps.apple.com/us/app/lockbook/id1526775001) [<img height= "30" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/78/Google_Play_Store_badge_EN.svg/2560px-Google_Play_Store_badge_EN.svg.png">](https://play.google.com/store/apps/details?id=app.lockbook)
+
 ![Discord](https://img.shields.io/discord/1014184997751619664?label=Discord&style=plastic)
 
 ## About
 _The private, polished note-taking platform._
 
-Privacy shouldn't be a compromise. That's why we made Lockbook, a companion for recording thoughts on all your devices. Record, sync, and share your notes with apps engineered to feel like home on every platform. We collect no personal information and encrypt your notes so even _we_ can’t see them. Don’t take our word for it: Lockbook is 100% open-source.
```

Comment:
```That's why we made Lockbook, a companion for recording thoughts on all your devices. Record, sync, and share your notes with apps engineered to feel like home on every platform```

the sentence feels a bit hard to digest at first. maybe split it into two sentences: one about how you can record sync, etc, your notes. and another on how lockbook makes you feel at home on every device your own cus we're multi-platform